### DISCUSSION TITLE
Show auth required or not with badge or icon

### DISCUSSION THREAD

**Discussion by channprj** [scalar/scalar#2435] [Ideas]
Hello 👋🏻

I'm a fan of scalar!

Previously I used swagger mainly, but scalar does not have a lock icon like below:

![swagger screenshot](https://github.com/scalar/scalar/assets/1831308/7fc32de7-2149-4412-a238-7d216486cff5)

How about adding a badge or icon for auth required or not?

![image](https://github.com/scalar/scalar/assets/1831308/ce38de79-1266-4271-9267-2a325e53d82d)

## Reference

- https://petstore.swagger.io

**Comment by hanspagel** [comment]
Yess, I’d love that!
